README.org

```diff
@@ -126,6 +126,7 @@
    mkdir aard2
    cd aard2
    git clone https://github.com/itkach/slobj.git
+   git clone https://github.com/itkach/slobber.git
```

Comment:
This repository does not use slobber, because it was not fine-tuned for Android. You can remove this line.